tests/test_models_semantic_segmentation.py

```diff
@@ -0,0 +1,451 @@
+import copy
```

Comment:
Same as above, if we can add these changes to the semantic segmentation that would be great. Just to help with the release on patch predictor.